code/ui/generalhud/inventorywrapper/effects/Effect.cs

```diff
@@ -38,39 +32,62 @@ private set
                 }
                 else
                 {
-                    Countdown?.Delete();
-                    Countdown ??= null;
+                    _countdownLabel?.Delete();
+                    _countdownLabel ??= null;
```

Comment:
Just htought about it and it really doesn't make much sense, as it does `_countdownLabel = _countdownLabel ?? null`

Reply:
This part was probably edited by me when refactoring names, so I didn't bother to look.